### PR TITLE
Readme install and licence

### LINKED ISSUE #29: installation process in Readme.md

### PR DESCRIPTION
**Pull request by hav3loc**
closes #29

### CODE REVIEW

**Comment by hav3loc** [comment]
Checked and verified.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,21 @@
 # Board&Roll
 
-Board&Roll is an application for finding and organizing board game meetings in your local area. Main goal is to help people who love board games.
+## Description
+**Board&Roll** is a web application for organizing and finding board game meetings in your local area. User can choose a game, create an event and invite friends or he can join one of the events nearby.
+
+
```
Comment (smt116) [inline README.md]: redundant empty line

File: README.md [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
+## Installation
+To download code from Github use command </br>
+`git clone https://github.com/Ragnarson-Internship2016/group-III.git ` </br>
```
Comment (smt116) [inline README.md]: `</br>`??

Comment (smt116) [inline README.md]: Imho it should not contain instruction how to clone repository. Just a sentence like "Clone repository from `...`"

File: README.md [hunk 3]
```diff
@@ -1,6 +1,21 @@
...
+Make sure that you already have bundler and nodejs.
+If you do not have it, use  
+```
+apt-get gem install bundler
+apt-get install nodejs
```
Comment (smt116) [inline README.md]:
```
~ λ apt-get gem install bundler
zsh: command not found: apt-get
```

1. Package manager is platform specific so it should only point that x and y are required for development.
2. `apt-get gem install bundler` is a valid command?

File: README.md [hunk 4]
```diff
@@ -1,6 +1,21 @@
...
+```
+To run application, open downloaded folder and run
+`rails s`.
```
Comment (smt116) [inline README.md]: This should be `rails server` (`s` is a shortcut). I would say "To run application, run `rails server` command in repository directory."

File: README.md [hunk 5]
```diff
@@ -1,6 +1,21 @@
...
+Then open your browser and go to `localhost:3000`
```
Comment (smt116) [inline README.md]: This is probably not needed as `rails server` output will include that info.

Comment (smt116) [inline README.md]: What about database and migrations?

File: LICENSE [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
```
Comment (trueskawka) [inline LICENSE]: shouldn't "express" be "expressed"?

Comment (Carolsien) [inline LICENSE]: MIT license was taken from https://opensource.org/licenses/MIT, so there should not be any mistakes

File: README.md [hunk 7]
```diff
@@ -11,5 +16,5 @@ Board&Roll is an application for finding and organizing board game meetings in y
 * [Karolina Rostek](https://github.com/Carolsien)
 * [Katarzyna Sikorska](https://github.com/Snieezy)
 * [Piotr Sordyl](https://github.com/hav3loc)
-
```
Comment (trueskawka) [inline README.md]: license is not displaying in a new line - check the View

**Comment by trueskawka** [comment]
+1

**Comment by gotar** [comment]
squash and you can merge

**Comment by gotar** [comment]
rebase with master

File: LICENSE [hunk 8]
```diff
@@ -0,0 +1,25 @@
+Copyright (c) 2016 Kamil Filipowski,
+                    Maciej Michalik,
+                    Bernard Pietraga,
+                    Alicja Raszkowska,
+                    Karolina Rostek,
+                    Katarzyna Sikorska,
+                    Piotr Sordyl
```
Comment (gotar) [inline LICENSE]: strange indentation

File: config/database.yml [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+# Warning: The database defined as "test" will be erased and
+# re-generated from your development database when you run "rake".
+# Do not set this db to the same as development or production.
+test:
+  <<: *default
+  database: db/test.sqlite3
+
+production:
+  <<: *default
+  database: db/production.sqlite3
```
Comment (gotar) [inline config/database.yml]: I was sure you have PG in project, why sqlite here?

**Comment by gotar** [comment]
why in the same comment when you change license you have other changes, like style for error pages

**Comment by hav3loc** [comment]
@gotar We are currently fixing it, we will change it to `ready to check` when we are done.

**Comment by hav3loc** [comment]
Fixed issues with unrelated files.

**Comment by gotar** [comment]
Squeeze to single commit

**Pull request merged**